src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(0.5),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chip: {
+      borderRadius: '4px',
+      color: 'white',
+      backgroundColor: ({ value }: { value: string }) => {
+        if (value === 'High') {
+          return secondary.main;
+        }
+
+        if (value === 'Medium') {
+          return warning.main;
+        }
+
+        return '#8f928b';
+      },
+    },
+  })
+);
+
+export interface SensitivityProps {
+  value: SensitivityType;
+}
+
+export const Sensitivity: FC<SensitivityProps> = (props) => {
+  const { icon, iconWrapper, chip } = useStyles(props);
+
+  return (
+    <Box>
+      <Box className={iconWrapper}>
```

Comment:
The `Box` component also defines a new css class for each component mounted, because of that I try to avoid them. Esp in cases where there could be lots of instance of the component on screen at once, like this.

So just replace with `<div>`.